src/test/java/org/opensearch/geospatial/GeospatialRestTestCase.java

```diff
@@ -160,7 +182,8 @@ protected Map<String, Object> getIndexMapping(String index) throws IOException {
         String responseBody = EntityUtils.toString(response.getEntity());
         Map<String, Object> responseMap = createParser(XContentType.JSON.xContent(), responseBody).map();
         MatcherAssert.assertThat(index + " does not exist", responseMap, Matchers.hasKey(index));
-        return (Map<String, Object>) ((Map<String, Object>) responseMap.get(index)).get(FIELD_MAPPINGS_KEY);
+        final Map<String, Object> mappings = (Map<String, Object>) ((Map<String, Object>) responseMap.get(index)).get(FIELD_MAPPINGS_KEY);
+        return (Map<String, Object>) mappings.get(MAPPING_PROPERTIES_KEY);
```

Comment:
Why do we change initial format by adding one more nested level?

Reply:
Created new method to extract properties.